src/contexts/OrderContext/index.js

```diff
@@ -1255,6 +1255,33 @@ export const OrderProvider = ({ Alert, children, strategy, isAlsea, isDisableToa
     })
   }
 
+  const handleLogEvent = async (events) => {
```

Comment:
I don't see where uses it this method, also you're returning nothing, remove the method if necessary.